src/components/expandable-help/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "@bcrs-shared-components/expandable-help",
+  "version": "0.0.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "vue": "^2.7.14"
+  },
+  "devDependencies": {
+    "vue-property-decorator": "^9.1.2"
```

Comment:
One of the things that we're going to have to get rid of in this repo 😟 . Here and in all the other components.

Reply:
Yup. We can do it all together (future).